clients/src/test/java/org/apache/kafka/clients/consumer/internals/DefaultBackgroundThreadTest.java

```diff
@@ -89,11 +89,20 @@ public void setup() {
     }
 
     @Test
-    public void testStartupAndTearDown() {
+    public void testStartupAndTearDown() throws InterruptedException {
         DefaultBackgroundThread backgroundThread = mockBackgroundThread();
         backgroundThread.start();
+
```

Comment:
Reviewer's note: we need to inject a small wait here since I moved the `running` flag from the constructor to the `run` method.